src/styles/blocks/stars.css

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  justify-content: space-around;
+  align-items: center;
+  height: 116px;
+  padding: 22px;
+  width: 140px;
+}
+
+.stars__inactive {
+  background-image: url(../../images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.stars__active {
+  background-image: url(../../images/star-active.svg);
+  height: 16px;
+  width: 16px;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
```

Comment:
The classes `.stars__inactive` and `.stars__active` are using `background-image` correctly, but the task requires using pseudo-selectors for yellow stars. Consider using `::before` or `::after` pseudo-elements to apply the background images for active and inactive stars.